packages/sourcecred/src/core/ledger/computeDistribution.js

```diff
@@ -21,11 +21,11 @@ import {type Distribution} from "./distribution";
  */
 export function computeDistribution(
   policies: $ReadOnlyArray<AllocationPolicy>,
-  accountsData: CredAccountData,
+  credGrainData: CredGrainView,
   effectiveTimestamp: TimestampMs
 ): Distribution {
   const allocationIdentities = _allocationIdentities(
```

Comment:
I'm wondering how much effort it would be to actually deprecate allocationIdentities as well, and instead just pass a (filtered-on-active) Array<ParticipantCredGrain>

Reply:
Can look into this, but I actually built the rest of the feature without this, so I would suggest finishing review and merge of the feature then prioritizing it as a separate refactor task.